lib/resque/scheduler/env.rb

```diff
@@ -43,7 +43,7 @@ def setup_backgrounding
         end
 
         Process.daemon(true, !Resque::Scheduler.quiet)
-        Resque.redis.client.reconnect
+        Resque.redis.redis._client.reconnect
```

Comment:
I know it is existing but `Resque.redis.redis` 😢

Reply:
Yea unfortunately is redis-namespace api to access the connection 😓 Maybe `#inner` would have been better? :rust-fan-boy: